tests/src/test/scala/system/rest/ControllerInfoSchemaTests.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2015-2016 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package system.rest
+
+import scala.util.Try
+
+import org.junit.runner.RunWith
+import org.scalatest.FlatSpec
+import org.scalatest.Matchers
+import org.scalatest.junit.JUnitRunner
+
+import com.jayway.restassured.RestAssured
+
+import common.WhiskProperties
+import spray.json.pimpString
+
+/**
+ * Basic tests of API info calls.
+ */
+@RunWith(classOf[JUnitRunner])
+class ControllerInfoSchemaTests extends FlatSpec with Matchers with RestUtil with JsonSchema {
+
+    it should "respond to GET / with general information as documented in swagger" in {
+
+        val auth = WhiskProperties.getBasicAuth
+        val response = RestAssured.
+            given().
+            config(sslconfig).
+            get(getBaseURL("/"))
```

Comment:
Note to self: Figure out how to use Akka-HTTP clients with modified ssl config to get rid of this thing.